package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "openmrs-module-ipd-frontend",
+  "version": "0.0.8",
+  "description": "React frontend for IPD module",
+  "main": "dist/index.js",
+  "author": "bahmni@thoughtworks.com",
+  "license": "AGPL-3.0",
```

Comment:
Change this to MPL-2.0 for this repository.

Reply:
Okay, will add